admin_honeypot/signals.py

```diff
@@ -1,3 +1,3 @@
-from django import dispatch
+from django.dispatch import Signal
 
-honeypot = dispatch.Signal()
+honeypot = Signal('request')
```

Comment:
I would try not to adjust imports if you don't absolutely need to. They make reading a pull request more difficult.

The big change here is adding a string argument named 'request' to the signal. But the [`Signal` class doesn't have an `__init__` argument where this would make any sense](https://github.com/django/django/blob/ef62a3a68c9d558486145a42c0d71ea9a76add9e/django/dispatch/dispatcher.py#L32):

```python
class Signal:
    ...

    def __init__(self, use_caching=False):
        ...
```

Can you explain a bit more about what you were trying to fix here?